pallets/collator-selection/src/migration.rs

```diff
@@ -0,0 +1,91 @@
+// Copyright Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+//! A module that is responsible for migration of storage for Collator Selection.
+
+use super::*;
+use frame_support::{log, traits::OnRuntimeUpgrade};
+
+/// Version 1 Migration
+/// This migration ensures that any existing `Invulnerables` storage lists are sorted.
+pub mod v1 {
+	use super::*;
+	#[cfg(feature = "try-runtime")]
+	use frame_support::inherent::Vec;
+	use frame_support::{pallet_prelude::*, weights::Weight};
```

Comment:
```suggestion
	use sp_std::prelude::*;
	use frame_support::pallet_prelude::*;
```

Weight is already in `pallet_prelude` afaik.